test/helpers/tokens.js

```diff
@@ -0,0 +1,47 @@
+
+const ERC20TokenList = async (params, from, tokenFactory) => {
+	/** 
+	 * Function returns list with deployed token.
+	 * 
+	 * @param {list|number} params list with token symbols || number of tokens you want to create
+	 * @param {signer} from signer of contract 
+	 *
+	*/
+
+	// Create list from token symbols. If params is an integer, an empty list is created.
+	params = typeof params === 'number' ? Array(params).fill({}) : params;
+	if (!Array.isArray(params)) params = [params];
+
+	// Formates an objects with keys for symbol and name from the list created above.
+	params = params.map((param, i) => {
+
+		if (typeof param === 'string') param = { symbol: param };
+		const args = Object.assign({}, { symbol: `TK${i}`, name: `Token ${i}`}, param);
+		if (typeof args === 'string') args = { symbol: args };
+		
+		let { name, symbol, decimals} = args;
+		if (!name) name = 'Token';
+		if (!symbol) symbol = 'TKN';
+		if (!decimals) decimals = 18;
+		return {
+			name: name,
+			symbol: symbol,
+			decimals: decimals,
+		};
+	});
```

Comment:
Do we ever actually need custom token symbols? If it's not really necessary, couldn't we just always create a list of tokens that have standard names and symbols sth like `names = token1, token2, token3...`, `symbols = TOK1, TOK2, TOK3...` and skip this complicated part of having a custom and a generic logic of creating the cosntructor parameters?